assets/src/dashboard/theme.js

```diff
@@ -161,12 +167,18 @@ const theme = {
     },
     tab: {
       family: 'Google Sans',
-      size: '14px',
+      size: 16,
       minSize: '12px',
       lineHeight: '20px',
       letterSpacing: '0.01em',
       weight: '500',
     },
+    smallLabel: {
+      family: 'Google Sans',
+      size: 12,
+      minSize: '12px',
+      letterSpacing: '0.01em',
+    },
```

Comment:
nit: I like the move away from `px` but I feel like we should have continuity with size and minSize and either include the units or not.